clients-src/modules/ROOT/partials/java/concepts/attribute_type.adoc

```diff
@@ -85,21 +85,21 @@ attributeType.asRemote(Transaction tx).getOwners(Set<TypeQLToken.Annotation> ann
 
 === Description
 
-Retrieve all Things that own an attribute of this type.
-Optionally, only fetches Things that own an attribute of this type with annotation set provided.
+Retrieve all Things that own an attribute of this type. +
+Optionally, filtered by annotations provided.
 
 === Input parameters
 
 [options="header"]
 |===
 |Name |Description |Type |Required |Default Value
 
-| Annotation
-| Only retrieve things that have an attribute of this type with the exact annotation set.
-a|
-* `TypeQLToken.Annotation.KEY`
-* `TypeQLToken.Annotation.UNIQUE`
-| False | N/A
+| annotations
+| Only retrieve things that have an attribute of this type with annotations (KEY or UNIQUE).
+| set of `Annotation`
+| False
+| Set<TypeQLToken.Annotation> annotations = Collections.emptySet();
```

Comment:
This should just be `Collections.emptySet()` or `emptySet()`

Reply:
Applies throughout the Java docs

Reply:
Fixed in https://github.com/vaticle/typedb-docs/pull/689/commits/b3b322c9f20475d0ec5ee2dcd4e0063adfc918ee